docs/how-to/upgrade-and-rollback-0.1.11-to-0.2.0.md

```
@@ -0,0 +1,96 @@
+## 4. Rollback Journey (0.2.0 → 0.1.11)
+
+If you must roll back to `0.1.11`:
+
+### Step 1: Reinstall 0.1.11
+```bash
+cargo install cargo-allow --version 0.1.11 --locked
+cargo-allow --version
+# Output: cargo-allow 0.1.11
+```
+
+### Step 2: Clean 0.2.0 Generated Artifacts
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)